dev-libs/zziplib/zziplib-0.13.71.recipe

```diff
@@ -103,11 +113,11 @@ INSTALL()
 		libzzipfseeko \
 		libzzipmmapped \
 		libzzipwrap
-
 	fixPkgconfig
 
 	packageEntries devel \
-		$developDir
+		$developDir \
+		$dataDir
```

Comment:
We should move `man3` to the `devel` package too.

Reply:
Also already in the local copy, should have pushed those changes